manage.py

```diff
@@ -71,7 +71,7 @@ def profile(length=25, profile_dir=None):
     app.run(ssl_context=('data/cert/server.crt', 'data/cert/server.crt'))
 
 @manager.command
-def runserver(host="127.0.0.1", port=8000):
+def runserver(host="127.0.0.1", port=8123):
```

Comment:
i made it a parameter `python manage.py runserver -p 8123` that defaults to `8008`